Lib/glyphsLib/builder/axes.py

```diff
@@ -520,10 +520,19 @@ def get_regular_master(font):
     """
     if not font.masters:
         return None
-    regular_name = font.customParameters["Variation Font Origin"]
-    if regular_name is not None:
+    # The current glyphs source specification supports the custom
+    # parameter name "Variable Font Origin".  This may have been
+    # named "Variation Font Origin" in the past.
+    # We support the current name with a fallback to the previous name
+    # if not found in the GSFont.customParameters dict
+    regular_id = None
+    if "Variable Font Origin" in font.customParameters:
+        regular_id = font.customParameters["Variable Font Origin"]
+    elif "Variation Font Origin" in font.customParameters:
+        regular_id = font.customParameters["Variation Font Origin"]
```

Comment:
you should expect a master name, not a UUID, here. then return the first master with that name

Reply:
https://github.com/googlefonts/glyphsLib/pull/587/commits/060c283df7ab91c42e8f3bc75605d7f21a09b1fd adds these changes and breaks the unit test

Reply:
https://github.com/googlefonts/glyphsLib/pull/587/commits/71419da74e106682aa3a053671284fae87155ec5 fixes the unit test which requires a hand edit of the test glyphs source file.  Note that the test file is a hand edited glyphs source file that changes the value of the custom parameter to the master name when  "Variation Font Origin" is used as the custom parameter name.  Previously I made the assumption that this key had an ID value and simply hand edited the source file key name to "Variation Font Origin" from "Variable Font Origin" using a source file exported from Glyphs @ build 2.6.4 (1286)

Reply:
and it looks like this fixes the other issue! :)